tests/test_mdnsbridge.py

```diff
@@ -41,9 +41,9 @@ def test_gethref_returns_first_service_with_matching_priority(self, rand, get):
         self.UUT.config['https_mode'] = "disabled"
 
         services = [
-            { "priority" : 100, "protocol" : "http", "address" : "service_address0", "port" : 12345, "hostname": "service_host0" },
-            { "priority" : 100, "protocol" : "http", "address" : "service_address1", "port" : 12346, "hostname": "service_host1" },
-            { "priority" : 100, "protocol" : "http", "address" : "service_address2", "port" : 12347, "hostname": "service_host2" },
+            { "priority" : 100, "protocol" : "http", "address" : "service_address0", "port" : 12345, "hostname": "service_host0", "versions": ["v1.0", "v1.1", "v1.2"] },
```

Comment:
This is probably personal taste due to these being tests, but could the versions not be tied to a variable to prevent the repetition? Also means the tests are a bit easier to update as later versions are added?
```suggestion
            versions = ["v1.0", "v1.1", "v1.2"] 
            { "priority" : 100, "protocol" : "http", "address" : "service_address0", "port" : 12345, "hostname": "service_host0", "versions": versions},
```

Reply:
Yes, agreed the default ones could certainly be parameterised. I'll sort that out.